ansible/roles/plpsql/templates/pg_hba.conf

```diff
@@ -92,9 +92,11 @@ host    all             all             ::1/128                 trust
 #host    replication     postgres        127.0.0.1/32            trust
 #host    replication     postgres        ::1/128                 trust
 
-# NB: `ssl` is neither supported nor enforced.
 {% if inventory_hostname == 'hkgmetadb.infra.ooni.io' %}
-host replication amsrepl {{ lookup('dig', 'amsmetadb.ooni.nu/A') }}/32 md5
+hostssl replication amsrepl {{ lookup('dig', 'amsmetadb.ooni.nu/A') }}/32 cert
```

Comment:
What CA is used to verify the client? As far as I see, `ssl_ca_file` is not set.

Also, CRL (`ssl_crt_file`) is not set, but revocation was not considered before as, given the low number of clients and lack of HA requirements, it may be easier to roll-over CA key in case of the key being compromised.

Reply:
Ah good point. I will add that.

I agree that revocation is currently out of scope.